File: proguardtest/app/src/main/java/cn/dozyx/proguardtest/app/MainActivity.java
package cn.dozyx.proguardtest.app;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import cn.dozyx.lib.EnumClass;
import cn.dozyx.lib.MyClass;

public class MainActivity extends AppCompatActivity {

    private EnumClass enumClass = EnumClass.Apple;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_proguard);

        MyClass myClass = new MyClass();
        myClass.doPublic("哈哈");


    }
}

File: proguardtest/lib/src/main/java/cn/dozyx/lib/ui/MainActivity.kt
package cn.dozyx.lib.ui

import android.app.Activity
import android.os.Bundle
import cn.dozyx.lib.MySubComplileOnlyClass
import cn.dozyx.lib.R

class MainActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_proguard)
        val clazz = MySubComplileOnlyClass()
        clazz.toString()
    }
}